presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -77,6 +93,12 @@ public static QueryRunner createQueryRunner(
         return createNativeQueryRunner(dataDirectory.get().toString(), prestoServerPath.get(), workerCount, cacheMaxSize, true);
     }
 
+    public static QueryRunner createJavaQueryRunner() throws Exception
+    {
+        String dataDirectory = System.getProperty("DATA_DIR");
```

Comment:
@tanjialiang Did you mean to use System.getenv similar to the existing createQueryRunner in this class?

Reply:
I came across this since it was throwing NPE even after setting the env variable as mentioned in the README file